test/cluster/test_cluster_config.py

```
@@ -0,0 +1,497 @@
+        return old_scylla_binary, ScyllaVersionDescription(path=str(old_scylla_binary), config={}, argv=[])
+
+    return Path(fallback.path), fallback
+
+
+CLUSTER_CONFIGS_QUERY = "SELECT configs FROM system_schema.scylla_clusters"
+KEYSPACE_CONFIGS_QUERY = "SELECT configs FROM system_schema.scylla_keyspaces WHERE keyspace_name = %s"
+TABLE_CONFIGS_QUERY = "SELECT configs FROM system_schema.scylla_tables WHERE keyspace_name = %s AND table_name = %s"
+
+
+async def wait_for_config_map_value(cql, host, query: str, params: list[str], config_name: str, expected_value: str | None) -> None:
+    async def configs_map_value_equal():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `configs_map_value_equal`

Add return type annotation: `bool | None`

(ANN202)